server/storage/mysql_test.go

```diff
@@ -22,15 +22,15 @@ func init() {
 		logrus.Fatal("MYSQL environment variable not set")
 	}
 
-	for i := 0; i < 30; i++ {
+	for i := 0; i <= 30; i++ {
 		gormDB, err := gorm.Open("mysql", dburl)
 		if err == nil {
 			err := gormDB.DB().Ping()
 			if err == nil {
 				break
 			}
 		}
-		if i == 29 {
+		if i == 30 {
```

Comment:
Yeah, I'm not overly bothered which it is in this case, just wanted to be clear the commit message isn't correct and it's a somewhat superfluous commit.